wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt

```diff
@@ -52,56 +60,66 @@ import org.dash.wallet.common.ui.dialogs.AdaptiveDialog
 import org.dash.wallet.common.ui.observeOnDestroy
 import org.dash.wallet.common.ui.viewBinding
 import org.dash.wallet.common.util.KeyboardUtil
+import org.dash.wallet.common.util.safeNavigate
+
+enum class ContactsScreenMode {
+    SEARCH_CONTACTS,
+    SELECT_CONTACT,
+    VIEW_REQUESTS
+}
 
 @AndroidEntryPoint
-class ContactsFragment : BottomNavFragment(R.layout.fragment_contacts_root),
+class ContactsFragment : Fragment(),
```

Comment:
`ContactsFragment` was added to the navigation graph and related changes were made.